docs/versioning.rst

```diff
@@ -0,0 +1,55 @@
+*****************
+Versioning Policy
+*****************
+
+The internal interface of the `ddtrace` library is defined as follows:
+    Any module, function, class or attribute is internal if it is prefixed with a single underscore
+
+    Any package that is named `internal` is internal
+
+    Any package, module, function, class or attribute that is contained within a namespace that is internal is also internal
```

Comment:
```suggestion
    Any package, module, function, class or attribute that is contained within a namespace that is internal is also internal.
```

Reply:
Leaving off the sentence punctuation as these term definitions are not sentences and punctation is not used elsewhere in this section.